catalog/ui/search-ui/standard/src/main/webapp/less/util.less

```diff
@@ -4,6 +4,7 @@
 }
 .padding-6{
   padding: 6px;
+  height: 40px;
```

Comment:
You'll want to put this in the filters.less instead of util.less.  

It should also go under it's own selector.  You can compose one using the existing html / classes, or you can add a new class to utilize.